src/js/line-chart.js

```diff
@@ -32,98 +52,87 @@ function LineChart () {
     draw: function () {
 
       var that = this
-        , margin = this.options.margin
-        , width = this.options.width  - margin.left - margin.right
-        , height = this.options.height - margin.top - margin.bottom
-        , data = this.options.data[0].values        // TODO: what if mulitple datasets...?
-        , columns = this.options.data[0].columns
-        , x
-        , y
-        , xColumn
-        , yColumn
-        , xDimension
-        , yDimension
-        , xAxis
-        , yAxis
+        , opt = this.options                                                  // just to shorten...
+        , margin = opt.margin
+        , width = opt.width  - margin.left - margin.right
```

Comment:
this.getCalculatedWidth(); eller et bedre navn, ligger i base utils, kanskje vi kan bruke dette